### DISCUSSION TITLE
Refactor function in nvim-cmp

### DISCUSSION THREAD

**Discussion by tranmanhlt7a4** [hrsh7th/nvim-cmp#1408] [Q&A] [answered]
Are there any ways to refactor something like a function, or variable name like IDE in nvim-cmp?
Or do I have to use another plugin? If so, can everyone suggest me some plugins? Thank you so much.

**Comment by fitrh** [comment, marked as answer]
cmp is a completion plugin, what you want is LSP rename capability and all you need is to call `vim.lsp.buf.rename`

**Reply by tranmanhlt7a4** [reply]
Thank you for your help 💖
